src/com/hms_networks/americas/sc/realtimedata/RealTimeTagDataPointManager.java

```diff
@@ -97,7 +97,7 @@ public int getSize() {
    * Gets a tag's current value into a list of tag current values.
    *
    * @param tag the tag to inspect
-   * @return returns a DataPoint object holding the tags current value
+   * @return a DataPoint object holding the tags current value
    */
   public void recordCurentTagValue(TagInfo tag) {

```

Comment:
This won't fix the Javadoc error you are seeing. That @ return is an invalid use of the @ return field. The method is a void, thus it returns nothing

Reply:
I was looking at the wrong function this whole time and still changed the right line. fixing that

Reply:
Why was a pull request made when you have not verified the fix?

Reply:
I should of changed the git submodule to pull from olwa-pr instead of master to verify. It seemed like a small thing

Reply:
this is tested. As a note I had to changed the github actions to not pull from master to test.